src/main/java/com/dezorganizacja/bridge/view/text/OrdersHandler.java

```diff
@@ -0,0 +1,58 @@
+package com.dezorganizacja.bridge.view.text;
+
+import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by maciek on 13.04.15.
+ */
+
+
+interface Command {
+    void runCommand(List<String> args);
+}
+
+public class OrdersHandler {
+    private final Map<String, Command> menuEntries = new HashMap<>();
+    private final Map<String, String> menuDetails = new HashMap<>();
+    {
+        menuEntries.put("h", this::help);
+        menuEntries.put("help", this::help);
+        menuEntries.put("credits", this::credits);
+        menuEntries.put("menu", this::print_menu);
+
+        menuDetails.put("start", "Start game");
+        menuDetails.put("end", "Quit game");
+        menuDetails.put("h", "Help system");
+        menuDetails.put("help", "Help system");
+        menuDetails.put("credits", "Credits");
+        menuDetails.put("menu", "Displays the possible prompt commands");
+    }
+
+    private void help(List<String> args) {
+        print("Some help should go here");
+    }
+
+    private void credits(List<String> args) {
+        print("Some credits should go here");
+    }
+
+    private void print_menu(List<String> args) {
+        for(String menuPosition : menuDetails.keySet()) {
+            print(menuPosition + " - " + menuDetails.get(menuPosition));
```

Comment:
lepiej by było, gdybyś wypisywał wszystkie aktywne komendy z opcjonalnymi opisami, dałoby to lepszy ogląd sytuacji, co możesz wykonać, a co nie, a opisy by były tylko pomocą, którą możnaby rozwijać wolniej, niż zbiór komend